zeros:
  torso_1_joint:                      0.0
  torso_2_joint:                      0.0
  head_1_joint:                       0.0
  head_2_joint:                       0.0
  arm_left_1_joint:                   0.0
  arm_left_2_joint:                   0.0
  arm_left_3_joint:                   0.0
  arm_left_4_joint:                   0.0
  arm_left_5_joint:                   0.0
  arm_left_6_joint:                   0.0
  arm_left_7_joint:                   0.0
  arm_right_1_joint:                  0.0
  arm_right_2_joint:                  0.0
  arm_right_3_joint:                  0.0
  arm_right_4_joint:                  0.0
  arm_right_5_joint:                  0.0
  arm_right_6_joint:                  0.0
  arm_right_7_joint:                  0.0
  hand_right_thumb_joint:             0.0
  hand_right_index_joint:             0.0
  hand_right_mrl_joint:               0.0
  hand_left_thumb_joint:              0.0
  hand_left_index_joint:              0.0
  hand_left_mrl_joint:                0.0

default_floating_base_pose:
  orientation:
    x: 0.
    y: 0.
    z: 0.
    w: 1.
  position: [0., 0., 1.2]
